src/lib/sedlexing.mli

```diff
@@ -22,6 +22,17 @@
     Of course, you'll probably want to define functions like [lexeme] to
     be used in the lexers semantic actions.  *)
 
+type position_info = {
+  position : int;
+  line : int;
+  line_position : int;
+}
+      (** Lexer position info structure. Fields:
+          * position: offset in stream
+          * line: number of newlines preceding offset
+          * line_position: offset since beginning of line
+          position, line, and line_position all start at 0 *)
+
```

Comment:
I think it would be better to just use `Lexing.position`. It's the de-facto standard, since the menhir interface is using it.

We could take an optional argument when creating the lexbuf for the file name and use a dummy one otherwise.